### PR TITLE
fix(doc): Explain how image deletion actually works

### PR DESCRIPTION
**Pull request by teor2345**
## Motivation

Image deletion keeps slightly more images than we want, which is ok for now, but we should document it.

## Review

Anyone can review this PR, we just need to fix the docs eventually.

### Reviewer Checklist

  - [ ] Docs make sense

### CODE REVIEW

File: .github/workflows/delete-gcp-resources.yml [hunk 1]
```diff
@@ -73,13 +73,15 @@ jobs:
 
       # Deletes cache images older than $DELETE_AGE_DAYS days.
       #
-      # Keeps the latest $KEEP_LATEST_IMAGE_COUNT images of each type:
+      # Keeps an extra $KEEP_LATEST_IMAGE_COUNT images of each type:
```
Comment (arya2) [inline .github/workflows/delete-gcp-resources.yml]:
```suggestion
      # Keeps an extra $KEEP_LATEST_IMAGE_COUNT of the most recent images that are older than $DELETE_AGE_DAYS of each type:
```

**Review by arya2 — COMMENTED** [review]
Makes sense

File: .github/workflows/delete-gcp-resources.yml [hunk 2]
```diff
@@ -73,13 +73,15 @@ jobs:
...
       # - zebrad checkpoint cache
       # - zebrad tip cache
       # - lightwalletd + zebrad tip cache
       #
-      # TODO: when we add testnet to the workflows, keep the latest $KEEP_LATEST_IMAGE_COUNT testnet images, 
-      #       and the latest $KEEP_LATEST_IMAGE_COUNT mainnet images.
+      # TODO:
+      # - keep the latest $KEEP_LATEST_IMAGE_COUNT, if there are that many recent images, delete all the outdated images
```
Comment (teor2345) [inline .github/workflows/delete-gcp-resources.yml]:
```suggestion
      # - keep the latest $KEEP_LATEST_IMAGE_COUNT, if there are at least that many recent images, delete all the outdated images
```

File: .github/workflows/delete-gcp-resources.yml (hunk 1 above)
Comment (teor2345) [reply .github/workflows/delete-gcp-resources.yml]: That's helpful, but maybe we could spell out exactly what happens:
```suggestion
      # Keeps all images younger than $DELETE_AGE_DAYS.
      # Also keeps $KEEP_LATEST_IMAGE_COUNT older images of each type:
```

**Review by gustavovalverde — APPROVED** [review]

**Pull request merged**